### DISCUSSION TITLE
GitHub Education - Pack

### DISCUSSION THREAD

**Discussion by YumeYuka** [community/community#144585] [GitHub Education] [answered]
### Hi everyone,

I applied for GitHub Education - Pack 3 days ago but it hasn't taken effect yet and I hope this is all going to be a little faster
![Screenshot_2024-11-15-07-35-56-657_com.microsoft.emmx-edit.jpg](https://img.nightrainmilkyway.cn/img/Screenshot_2024-11-15-07-35-56-657_com.microsoft.emmx-edit.jpg)

**Comment by davevad93** [comment, marked as answer]
Hi @NightRainMilkyWay ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @NightRainMilkyWay was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
